e2e/cypress/support/okta_commands.js

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {_} from 'lodash';
```

Comment:
Was this used somewhere? In case yes, you may use `Cypress._.`

Reply:
leftover from some previous iteration, removed.